src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -43,10 +48,26 @@ public static PreSubmitCallbackResponse<SscsCaseData> processPostponementRequest
             return response;
         }
 
+        Hearing latestHearing;
+        try {
+            latestHearing = sscsCaseData.getLatestHearing();
+        } catch (NullPointerException e) {
```

Comment:
You shouldn't be catching and rethrowing the NPE as a RTE

Reply:
Agreed, though I see why he did it. He wanted to put the case id in the error.

The case id should probably be reported via logging, which should probably be handled by AOP if at all possible.

I'll create a commit to remove this rethrow and see if there's an opportunity to add some temporary logging for the case id to compensate for the loss of information while we don't have AOP.